sdk/head_tracker.cc

```diff
@@ -22,6 +22,65 @@
 #include "util/vectorutils.h"
 
 namespace cardboard {
+namespace {
+
+void GetViewporOrientationRotation(
+        const CardboardViewportOrientation ViewportOrientation,
+        Rotation* ekf_to_head_tracker,
+        Rotation* sensor_to_display) {
+
+  // This is the same than initializing the rotation from
+  // Rotation::FromYawPitchRoll(-M_PI / 2., 0, -M_PI / 2.).
+  static const Rotation kEkftoHeadTrackerLandscapeLeft =
+          Rotation::FromQuaternion(Rotation::QuaternionType(0.5, -0.5, -0.5, 0.5));
+  static const Rotation kSensorToDisplayLandscapeLeft =
+          Rotation::FromAxisAndAngle(Vector3(0, 0, 1), M_PI / 2.);
```

Comment:
Please use float literals (see https://en.cppreference.com/w/cpp/language/floating_literal). Plese fix here and there.